### PR TITLE
Remove Default sku setting from NewAzureVirtualNetworkGateway commandlet

### PR DESCRIPTION
**Pull request by ritwikbasu**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by praries880** [comment]
@MikhailTryakhov can you look at this PR?

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs [hunk 1]
```diff
@@ -265,19 +265,8 @@ private PSVirtualNetworkGateway CreateVirtualNetworkGateway()
...
-                    vnetGateway.Sku.Name = MNM.VirtualNetworkGatewaySkuTier.VpnGw1;
-                }
-                else
-                {
-                    vnetGateway.Sku = new PSVirtualNetworkGatewaySku();
-                    vnetGateway.Sku.Tier = MNM.VirtualNetworkGatewaySkuTier.Basic;
-                    vnetGateway.Sku.Name = MNM.VirtualNetworkGatewaySkuTier.Basic;
-                }
+                // If gateway sku param value is not passed,  - Let NRP make the decision, just pass it as null here
+                vnetGateway.Sku = null;
```
[the remark this answers is not in the document]
Comment (ritwikbasu) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: @praries880 - The test is being re-recorded. Should be done in the next couple of hours.

**Comment by maddieclayton** [comment]
@ritwikbasu Please also open a PR targeting this branch: https://github.com/Azure/azure-powershell/tree/Network_preview.  We will do the out of band release from here.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (praries880) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: @ritwikbasu Thanks :)

[the remark this answers is not in the document]
Comment (ritwikbasu) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: Added re-recorded test case.

**Comment by maddieclayton** [comment]
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/899/

**Pull request merged**